### PR TITLE
Remove media_entity_document because it is not used by Open Y. We migrated to core media

### PR DESCRIPTION
**Pull request by podarok**
See https://github.com/ymcatwincities/openy/wiki/Important-versions-for-upgrade-path

## media_ modules overview
There are following `media_` modules in Open Y:
- **media_directories** - this is a module that provides Media Library browser thing, should be kept untouched;
- **media_entity** - legacy media_entity module, moved to Drupal Core, left in composer.json as a dependency for the Upgrade path from Open Y 1.x;
- **media_entity_document** - is not used by Open Y anymore, left in composer.json as a dependency for the Upgrade path from Open Y 1.x.
- **media_entity_image** - is not used by Open Y anymore, left here for the Upgrade path from Open Y 1.x;
- **media_entity_video** - this is a module in use. There are a couple of issues to resolve to get rid of the module: [fix d9 compatibility](https://www.drupal.org/project/media_entity_video/issues/3148060), [migrate to Drupal Core media](https://www.drupal.org/project/media_entity_video/issues/3149789) - both addressed in this PR. The module can be marked as deprecated in the release note. Should be kept in composer.json for the Upgrade path from Open Y 1.x.

Original Issue, this PR is going to fix: remove not used module in Q2 2021

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [ ] Check before this PR module is disabled
image

- [ ] Verify media_entity_document module not being pulled by composer.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2926/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2926/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2926/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2926/

**Comment by podarok** [comment]
image

**Review by hamrant — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -107,7 +107,6 @@
         "oomphinc/composer-installers-extender": "^1.1 || ^2.0.0",
         "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2",
         "symfony/process": "^4",
-        "ymcatwincities/media_entity_document": "*",
```
Comment (podarok) [inline composer.json:113]:
```suggestion
"ymcatwincities/media_entity_document": "*", new code
```

**Review by podarok — COMMENTED** [review]
for @duozersk

File: composer.json (hunk 1 above)
Comment (duozersk) [inline composer.json:113]:
```suggestion
testing it out
```

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3185/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3185/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3185/

**Pull request merged**